fbcv-simple.obo

```diff
@@ -10227,10 +10226,28 @@ is_obsolete: true
 [Typedef]
 id: FBcv:0008000
 name: has_age
-namespace: FlyBase_miscellaneous_CV
 is_functional: true
 
+[Typedef]
+id: RO:0002314
+holds_over_chain: inheres_in BFO:0000050
+
```

Comment:
why does RO:0002314 not have a name?
(it is 'inheres in part of')

Reply:
I am not sure why, but my feeling is that it will appear one we next run it.. you could try and run it again; maybe it has to do with the imports not being updated.

Reply:
just ran again (sh run_release.sh) on this branch and still no name - will keep an eye on it when I run through the full set of releases, perhaps it will fix after doing fbdv and dpo again.

Reply:
Since this is not a formal error, lets roll with it; But make a ticket so we monitor it for next time.